packages/panels/resources/views/components/sidebar/index.blade.php

```diff
@@ -34,7 +34,7 @@
         @endif
     @endif
     @class([
-        'fi-sidebar fixed inset-y-0 start-0 z-30 grid h-screen content-start bg-white transition-all dark:bg-gray-900 lg:z-0 lg:bg-transparent lg:shadow-none lg:ring-0 lg:transition-none dark:lg:bg-transparent',
+        'fi-sidebar fixed inset-y-0 start-0 z-30 grid h-screen content-start bg-white transition-all dark:bg-gray-900 lg:bg-transparent lg:shadow-none lg:ring-0 lg:transition-none dark:lg:bg-transparent',
```

Comment:
I feel like there must be a reason why `lg:z-0` was used initially... Have you tested all sidebar configurations (collapsible, fully collapsible, etc.)?

Reply:
@zepfietje yes I did, I also provided screenshots for the different sidebar states/modes, including when a modal is active.

I am unsure what the purpose of removing z-index in larges screens is for. I looked up the commit history and it goes all the way back to 2.x branch.

Reply:
Hmm alright, let's hope it doesn't break anything then...